.github/workflows/rust.yml

```diff
@@ -2,7 +2,8 @@ name: Rust
 
 on:
   push:
-    branches: [ main, 'devnet_*', 'testnet_*' ]
+    branches: [ 'devnet_*', 'testnet_*' ]
```

Comment:
Does this mean that we'll still run these tests on every commit in PRs to devnet/testnet branches?